docs/kb/semgrep-ci/scan-monorepo-in-parts.md

```diff
@@ -42,19 +42,19 @@ For the other modules, the commands look similar. For module B:
 
 You will then have the flexibility to trigger each one on appropriate events or frequencies.
 
-Now that you've successfully configured your monorepo to be scanned in parts, you also have to configure the findings from each part or module to show up as their own project in Semgrep AppSec Platform.
+Now that you understand how to configure your monorepo to be scanned in parts, you also have to understand how to configure the findings from each part or module to show up as their own project in Semgrep AppSec Platform.
 
-To ensure findings from the module are assigned to their own project in Semgrep AppSec Platform, you will need to explicitly set the `SEMGREP_REPO_NAME` environment variable ([see CI environment variables reference](/docs/semgrep-ci/ci-environment-variables/#semgrep_repo_name)).
+To assign findings from the module to their own project in Semgrep Appsec Platform, you must explicitly set the `SEMGREP_REPO_DISPLAY_NAME` environment variable ([see CI environment variables reference](/docs/semgrep-ci/ci-environment-variables#semgrep_repo_display_name)).
```

Comment:
I see that in the link we mention the minimum version of Semgrep required - in case folks don't click through, I wonder if we should make a quick note of that here as well.